packages/teams-js/src/public/stageView.ts

```diff
@@ -34,6 +34,11 @@ export namespace stageView {
      */
     threadId: string;
 
+    /**
+     * The messageId identifies a particular channel meeting within the channel as specified by the threadId above.
+     */
+    messageId?: string;
```

Comment:
is there any format for what we expect `messageId` to look like or can it be any string e.g `"apps"`?

Reply:
Can it be an empty string, for example? If not you should check for that and not allow it to be passed up to the host

Reply:
Its mostly a numeric string. Pretty similar to what we have in the threadId (the parameter above the messageId)

Reply:
Is there any way for someone using the `messageId` improperly that would cause errors? Should we consider creating a separate type/class for this kind of `id` to ensure correctness.

Reply:
If the messageId thats passed doesnt point to an existing meeting then we will fail it on the client. Similar to threadId